### ISSUE TITLE
Collisions between hand and object meshes during trimesh visualization

### ISSUE THREAD

**Issue by RbtcsOscar** [UVR-WJCHO/HOGraspNet#5]
Hi,
Thank you for sharing your great work and the special datasets.

While using your dataset for my research, I encountered a problem during visualization.
In your visualization.py code, I added a simple 3D visualization using trimesh to check how the grasp poses look. However, I found that there are many collisions between the hand meshes and object meshes, as shown in the image below:

<img width="914" height="576" alt="Image" src="https://github.com/user-attachments/assets/746ad045-144b-42f1-98f9-44f9069ea3b1" />

Here is the additional code I used for visualization:

        scene = trimesh.Scene()

        obj_mesh = trimesh.Trimesh(vertices=obj_verts_world.squeeze().cpu().detach().numpy(), faces=obj_faces_template.squeeze().cpu().detach().numpy())
        hand_mesh = trimesh.Trimesh(vertices=mano_verts.squeeze().cpu().detach().numpy(), faces=hand_faces_template.squeeze().cpu().detach().numpy())

        scene.add_geometry([obj_mesh, hand_mesh])
        scene.show()

Do you have any suggestions or possible reasons why these collisions might occur?
Thank you very much for your help!

Best regards,
By Oscar

**Comment by UVR-WJCHO** [comment]
Hello,
Please accept our sincere apologies for the significantly delayed reply. Due to the author's personal circumstances, maintaining and updating this GitHub repository has been difficult for some time.
Although this response comes too late to be useful, please reach out to us at woojin.cho@kaist.ac.kr if you are still in need of any further information.
